mvc/src/main/java/com/interface21/webmvc/servlet/view/JsonView.java

```diff
@@ -1,14 +1,35 @@
 package com.interface21.webmvc.servlet.view;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.interface21.webmvc.servlet.View;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
-
+import java.io.PrintWriter;
 import java.util.Map;
 
 public class JsonView implements View {
 
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+    private static final String APPLICATION_JSON_UTF8_VALUE = "application/json;charset=UTF-8";
+
     @Override
     public void render(final Map<String, ?> model, final HttpServletRequest request, HttpServletResponse response) throws Exception {
+        response.setContentType(APPLICATION_JSON_UTF8_VALUE);
+        response.setCharacterEncoding("UTF-8");
+
+        final Object responseData = determineResponseData(model);
+        final String jsonString = objectMapper.writeValueAsString(responseData);
+
+        try (final PrintWriter writer = response.getWriter()) {
+            writer.write(jsonString);
+            writer.flush();
+        }
```

Comment:
```suggestion
        try (final PrintWriter writer = response.getWriter()) {
            writer.write(jsonString);
            writer.flush();
        } catch (JsonProcessingException e) {
            response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
            response.getWriter().write("{\"error\":\"JSON 변환 실패\"}");
        }
```

objectMapper가 JSON으로 직렬화하거나 역직렬화할 때 문제가 발생할 경우, 500 상태 코드와 오류메시지를 전송하도록 catch를 추가하는 것은 어떨까요?